### ISSUE TITLE
Prop mods are broken (libdogma 1.2.0 yc119.1)

### ISSUE THREAD

**Issue by Artefact2** [osmium-org/osmium#184]

**Comment by Artefact2** [comment]
https://o.smium.org/db/effect/6730
https://o.smium.org/db/effect/6731

This is not a bug, the expressions have been removed. I'll have to manually write expressions for those in libdogma.

Is there some math on the speed boost calculation somewhere?

**Comment by Artefact2** [comment]
https://oldforums.eveonline.com/?a=topic&threadID=141812

~~~
= Cruising speed * (1 + (base boost of AB or MWD * (1 + accel control level * 0.05) * (ab/mwd implant(s)) * (thrust / (ship mass + modules mass)))
~~~

**Comment by Artefact2** [comment]
Fixed in libdogma 493a7c99e99df0eb12f4fd2c033a8a054100a9d9.
